### PR TITLE
chore: upgrade to alpine 3.19

### PR DESCRIPTION
**Pull request by misiek08**
We wanted to use newest ffmpeg. Without switching to ubuntu static builds are not properly resolving DNS (glibc vs musl), so the easiest way is to upgrade alpine.

### CODE REVIEW

**Comment by misiek08** [comment]
Not sure if `feat` or `chore`

File: CMakeLists.txt [hunk 1]
```diff
@@ -9,6 +9,9 @@
 # Minimum CMake version.  This must be in the root level CMakeLists.txt.
 cmake_minimum_required(VERSION 3.16)
 
+# Musl define off_t and it's already 64bit
+set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -Doff64_t=off_t")
```
Comment (joeyparrish) [inline CMakeLists.txt:13]: I don't think this is universally true on all platforms.  We shouldn't assume off_t is 64-bits, even if it's true in your case.

**Review by joeyparrish — CHANGES_REQUESTED** [review]

**Comment by joeyparrish** [comment]
Please merge with `main`, which has changed a lot in the recent `cmake` port.  Thanks!

File: CMakeLists.txt (hunk 1 above)
Comment (joeyparrish) [reply CMakeLists.txt:13]: It appears that the use of the non-standard off64_t in absl was fixed upstream in https://github.com/abseil/abseil-cpp/commit/4500c2fada4e952037c59bd65e8be1ba0b29f21e .  Upgrading absl should resolve this.  Testing now.

Comment (joeyparrish) [reply CMakeLists.txt:13]: Resolved by #1338 and #1337

**Review by cosmin — APPROVED** [review]

**Review by joeyparrish — APPROVED** [review]

**Pull request merged**